app/src/test/java/com/x8bit/bitwarden/ui/vault/feature/item/util/VaultItemTestUtil.kt

```diff
@@ -75,9 +75,9 @@ fun createIdentityView(isEmpty: Boolean): IdentityView =
 
 fun createSshKeyView(isEmpty: Boolean): SshKeyView =
     SshKeyView(
-        privateKey = "privateKey".takeUnless { isEmpty },
-        publicKey = "publicKey".takeUnless { isEmpty },
-        fingerprint = "fingerprint".takeUnless { isEmpty },
+        privateKey = "privateKey".takeUnless { isEmpty } ?: "",
+        publicKey = "publicKey".takeUnless { isEmpty } ?: "",
+        fingerprint = "fingerprint".takeUnless { isEmpty } ?: "",
```

Comment:
So is the `SshKeyView` requiring the values to be nonnull too?

Reply:
It is

Reply:
I see, can we use `orEmpty()`?